### ISSUE TITLE
Add flatpak support.

### ISSUE THREAD

**Issue by Anikmoujegy** [chrismaltby/gb-studio#1286]
I will be great if you add flatpak support. Then it will be easy to get it on Linux.

**Comment by patrickmollohan** [comment]
Linux has .deb, .rpm, and .AppImage support, so it is incredibly easy to get it on Linux. Also, given Flatpak's association with Red Hat, I don't think it's a good idea to support their shadiness in any way, shape, or form. Just my opinion.

**Comment by toonn** [comment]
Flatpak is way more broadly useful than just on RHEL.
[There is a package on FlatHub](https://flathub.org/apps/dev.gbstudio.gb-studio) but I assume that's unofficial.

**Comment by patrickmollohan** [comment]
I agree, it is. However, given Flatpak was created by RHEL employees, I have no interest in it. I don't run the show here, I just contribute from time to time, but as the person who added .AppImage support, I can assure you I won't be putting in the effort for Flatpak.
